airmozilla/main/views.py

```diff
@@ -1114,6 +1115,90 @@ def item_pubdate(self, event):
         return event.start_time
 
 
+def related_content(request, slug):
+    event = get_object_or_404(Event, slug=slug)
+
+    es = pyelasticsearch.ElasticSearch(settings.RELATED_CONTENT_URL)
+    mlt_query = {
+        "more_like_this": {
+            "fields": ["title", "tags", "channels"],
```

Comment:
Make this a variable. A couple of lines up here in this function. Then, use that to NOT include `channels` if the event you're on is only in the default channel. E.g. something like this:
```python
fields = ['title', 'tags']
if list(event.channels.all()) != [Channel.objects.get(slug=settings.DEFAULT_CHANNEL_SLUG):
    fields.append('channel')
```
Obviously, you might need to format that differently so it fits better.